pkg/kubelet/kubelet_pods.go

```diff
@@ -142,15 +142,15 @@ func makeMounts(pod *v1.Pod, podDir string, container *v1.Container, hostName, h
 	// Kubernetes will not mount /etc/hosts if:
 	// - when the Pod sandbox is being created, its IP is still unknown. Hence, PodIP will not have been set.
 	mountEtcHostsFile := len(podIPs) > 0 && runtime.GOOS != "windows"
-	klog.V(3).Infof("container: %v/%v/%v podIPs: %q creating hosts mount: %v", pod.Namespace, pod.Name, container.Name, podIPs, mountEtcHostsFile)
+	klog.V(3).InfoS("Container is creating hosts mount", "pod", klog.KObj(pod), "containerName", container.Name, "podIPs", podIPs, "mountEtcHostsFile", mountEtcHostsFile)
```

Comment:
Let's keep the names consistant if they refer to the same type, `mountEtcHostsFile` is just a path
```suggestion
	klog.V(3).InfoS("Container is creating etc hosts mount", "pod", klog.KObj(pod), "containerName", container.Name, "podIPs", podIPs, "path", mountEtcHostsFile)
```